src/program_generator/pg_marginal.py

```diff
@@ -96,7 +96,7 @@ def _do_generate(self, opcode, max_op_count, shuffle_counts, step_op_count):
 
   def _generate_single_program(self, operation, op_count):
     # for compatibility with the generate_single_marginal function
-    single_op_pushes = ["60a1"] * arity(operation)
+    single_op_pushes = ["6003"] * arity(operation)
```

Comment:
this isn't related to PUSHx that much, but this is related to the pivot we need to make related to the pg_arguments models. Nevermind for now. In general this is a small-non-zero argument we'll use in pg_marginal